Packs/FeedMISPThreatActors/Integrations/FeedMISPThreatActors/test_data/misp_threat_actor_galaxy_example.json

```diff
@@ -0,0 +1,28 @@
+{
+    "authors": [],
+    "category": "actor",
+    "description": "Known or estimated adversary groups targeting organizations and employees. Adversary groups are regularly confused with their initial operation or campaign. threat-actor-classification meta can be used to clarify the understanding of the threat-actor if also considered as operation, campaign or activity group.",
```

Comment:
Threat-actor-classification meta can be used to clarify the understanding of the threat-actor, if also considered as operation, campaign or activity group.",